db/migrate/20210407003612_change_default_valuefor_allows_greeters.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeDefaultValueforAllowsGreeters < ActiveRecord::Migration[6.0]
+  def change
+    change_column_default(:vita_partners, :allows_greeters, from: nil, to: true)
```

Comment:
I'm surprised you want the default to be `true`. Is that OK with product?

Reply:
Yeah I asked Kelly about this and she said it should default to true. It will still be a checkbox option on the create org page but will be selected upon the page load. Since greeters are currently greeting for almost every organization, except for a few that have requested no greeters I think it makes sense to set true as the default.